lib/svg.dart

```diff
@@ -450,6 +462,10 @@ class SvgPicture extends StatefulWidget {
   /// will take the height of its parent.
   final double height;
 
+  final BoxFit fit;
```

Comment:
/// How to inscribe the picture into the space allocated during layout.  The default is [BoxFit.contain].